c7n/utils.py

```diff
@@ -324,6 +325,17 @@ def __contains__(self, other):
 worker_log = logging.getLogger('c7n.worker')
 
 
+def read_aws_config_file(file=None):
+    """ Read the AWS config file and return the config """
+
+    if file is None:
+        file = os.path.expanduser(os.path.join('~', '.aws', 'config'))
```

Comment:
Do we support Windows? If not then we can just say `~/.aws/config`. If so then we should check `%UserProfile%\.aws` when on Windows. In either case we should also honor `AWS_CONFIG_FILE` as an override.

http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-config-files